src/shared/models/StructuredNotification.ts

```diff
@@ -0,0 +1,21 @@
+type StructuredNotification = {
```

Comment:
Are all these fields really optional? Can we make some of the known ones required?

Reply:
we could probably make `id` `heading` and `content` required. the other ones I'm not 100 percent sure on since the `rawNotification` is of `any` type

Reply:
Ya, I think the two things we know for sure is `id` and `content` will be in the payload.

Reply:
Fixed